solr/solrj/src/java/org/apache/solr/client/solrj/io/stream/CloudSolrStream.java

```diff
@@ -491,6 +490,11 @@ public boolean equals(Object o) {
       return this == o;
     }
 
+    @Override
```

Comment:
As someone who has at least a passing understanding of this code, this and the other Streaming class implementations below gets my +1.

Reply:
OK, thanks.